tests/plugins/test_koji_upload.py

```diff
@@ -19,7 +19,7 @@
     import koji
 except ImportError:
     import inspect
-    import sys  # noqa: F811
```

Comment:
Why this cannot be removed?

Reply:
It could, but it seemed that someone specifically went out of their way to disable that flake8 warning, so I assumed it was intentional (maybe the whole block is supposed to be sort of autonomous, without depending on whether `sys` is already imported or not?)

Reply:
I've no idea why :)  I'd just remove that redefinition